exporter/elasticsearchexporter/README.md

```diff
@@ -516,3 +519,16 @@ This gives the exporter the opportunity to group all related metrics into the sa
 
 2. Otherwise, check your metrics pipeline setup for misconfiguration that causes an actual violation of the [single writer principle](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#single-writer).
  This means that the same metric with the same dimensions is sent from multiple sources, which is not allowed in the OTel metrics data model.
+
+### flush failed (400) illegal_argument_exception
```

Comment:
Awesome, thanks for adding this. It should make it easier for users affected by this change to fix their setup.